hidi/matrix.py

```diff
@@ -226,6 +227,74 @@ def transform(self, M,  **kwargs):
         return self.keras_model, kwargs
 
 
+class KerasKfoldTransform(Transform):
+    """
+    Generalized transform for Keras algorithm with k fold cross validation
+    evaluation
+    """
+    def __init__(self, keras_model, validation_matrix, tensorboard_log_dir,
```

Comment:
Let's remove this function altogether and leave it to the user to configure everything for Keras callbacks. If we start adding built in functionality for it, transforms will get more and more complicated as we make more and more complex keras models.